backend/src/database/seeders/20220713115148-seed-users.js

```diff
@@ -1,9 +1,12 @@
+// eslint-disable-next-line @typescript-eslint/no-var-requires
+const bcrypt = require('bcrypt');
+
 module.exports = {
-  up(queryInterface) {
+  async up(queryInterface) {
     return queryInterface.bulkInsert('Users', [
       {
         email: 'admin@rasporedar.com',
-        password: 'admin',
+        password: await bcrypt.hash('admin', parseInt(process.env.SALT_ROUNDS)),
```

Comment:
Nitpick: this hashing does not have the same default as the one in the user model. Maybe extract it to a common method.